esmvaltool/diag_scripts/ocean/diagnostic_maps.py

```diff
@@ -26,11 +29,16 @@
 import logging
 import os
 import sys
+from itertools import product
+import matplotlib
+matplotlib.use('Agg')  # noqa
```

Comment:
This line is no longer needed, this is now set by the backend via an environmental variable.

Reply:
Hi @bouweandela, thanks for the help. 

This line is not needed Inside the main body of ESMValTool. However, if you're running the diagnostic standalone, then it won't work without this line (for me).

ie: this command won't work without the `matplotlib.use('Agg') `line:
```
2019-01-28 13:14:31,348 UTC [29511] INFO    To re-run this diagnostic script, run:
MPLBACKEND="Agg" $conda/envs/esmvaltool/bin/python $ESMValTool/esmvaltool/diag_scripts/ocean/diagnostic_seaice.py $run/GlobalOcean/recipe_ocean_ice_extent_20190128_131305/run/diag_ice_SHW/Global_seaice_timeseries/settings.yml
```

Reply:
What shell are you using? The command works in bash. In e.g. (t)csh you would need to write
```tcsh
set MPLBACKEND="Agg"; $conda/envs/esmvaltool/bin/python $ESMValTool/esmvaltool/diag_scripts/ocean/diagnostic_seaice.py $run/GlobalOcean/recipe_ocean_ice_extent_20190128_131305/run/diag_ice_SHW/Global_seaice_timeseries/settings.yml
```

Reply:
See also https://matplotlib.org/faq/environment_variables_faq.html#envvar-MPLBACKEND

Reply:
Okay, that works! I'll remove the rest of the backend lines, cheers!